pkg/sql/plan2/function/operators.go

```diff
@@ -39,6 +39,446 @@ func initOperators() {
 // operators contains the operator function indexed by function id.
 var operators = map[int][]Function{
 	// comparison operator
+	IS: {
+		{
+			Index:  0,
+			Flag:   plan.Function_STRICT,
+			Layout: COMPARISON_OPERATOR,
+			Args: []types.T{
+				types.T_bool,
+				types.T_bool,
+			},
+			ReturnTyp:   types.T_bool,
+			TypeCheckFn: strictTypeCheck,
+			Fn:          operator.Is,
+		},
+	},
+	ISNOT: {
```

Comment:
input two booleans then returns a boolean, 
you sure this is the correct semantic for IS/ISNOT ?

Reply:
Yes.
sql: [exp/value] IS/IS NOT **[TRUE/FALSE]**

The value in left should be calculated/casted as a boolean value.
And the value in right must be TRUE/FALSE.
Therefore IS/IS NOT just accept boolean or it will return error